src/Collection.php

```diff
@@ -39,10 +39,14 @@ public function __construct(
      */
     public function findTargetClasses(string $attribute): array
     {
-        return array_map(
-            fn(array $a) => new TargetClass(new $attribute(...$a[0]), $a[1]),
-            $this->targetClasses[$attribute] ?? []
-        );
+        try {
+            return array_map(
+                fn(array $a) => new TargetClass(new $attribute(...$a[0]), $a[1]),
+                $this->targetClasses[$attribute] ?? []
+            );
+        } catch(\Throwable $e) {
+            throw new \LogicException("'Error creating attribute [$attribute].", previous: $e);
```

Comment:
```suggestion
            throw new \LogicException(sprintf('Error creating attribute "%s".', $attribute), previous: $e);
```
It's better to wrap the word in quotes instead of brackets. `sprintf` is quiet good way not to do quotes escaping.